docs/ad/index.md

```diff
@@ -26,10 +24,17 @@ To get started, choose **Anomaly Detection** in Kibana.
 A detector is an individual anomaly detection task. You can create multiple detectors, and all the detectors can run simultaneously, with each analyzing data from different sources.
 
 1. Choose **Create Detector**.
-2. For **Define detector**, enter the **Name of the detector** and a brief **Description**. Make sure the name that you enter is unique and descriptive enough to help you to identify the purpose of this detector.
-3. For **Data source**, choose the index that you want to use as the data source. You can optionally use index patterns to choose multiple indices. Choose the **Timestamp field** in your index.
-4. For **Filter Data**, you can optionally filter the index that you chose as the data source. From the **Filter type** menu, choose  **Visual filter**, and then design your filter query by selecting **Fields**, **Operator**, and **Value**, or choose **Custom Expression** and add in your own JSON filter query.
-5. Choose **Create**.
+1. Enter the **Name** of the detector and a brief **Description**. Make sure the name that you enter is unique and descriptive enough to help you to identify the purpose of this detector.
+1. For **Data source**, choose the index that you want to use as the data source. You can optionally use index patterns to choose multiple indices.
+1. Choose the **Timestamp field** in your index.
+1. For **Data filter**, you can optionally filter the index that you chose as the data source. From the **Filter type** menu, choose  **Visual filter**, and then design your filter query by selecting **Fields**, **Operator**, and **Value**, or choose **Custom Expression** and add in your own JSON filter query.
+1. For **Detector operation settings**, define the **Detector interval** to set the time interval at which the detector collects data. The shorter you set this interval, the more data points the detector gets, but the more computing resources it consumes.
```

Comment:
Detector will aggregate data during the interval, then feed the aggregated result into model. So the shorter interval means detector will aggregate less data points. 
AD model shingle process needs at least 6 aggregated data points in 8 continuous intervals. So a shorter interval means the model will pass shingle process more quickly and start to generate anomaly result.

Reply:
Added.